data/human/human missions.txt

```diff
@@ -5428,3 +5428,66 @@ mission "Deal Change 2"
 				die
 			label skip
 			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`
+
+
+
+government "Escort (Betraying)"
+	"display name" "Merchant"
+	swizzle 5
+	
+	"player reputation" 10
+	"attitude toward"
+		"Korath" -.01
+	"fine" 0
+	"friendly hail" "friendly civilian"
+	"hostile hail" "hostile pirate"
+
+mission "Escort offer (Betrayal)"
+	landing
+	name "Escort to <planet>"
+	description "The captain of the <npc> has offered to fly alongside you, protecting you from pirates, on your way to <destination>. In exchange, he has asked for 80,000 credits upon your arrival."
+	source Sunracer
+	destination Maelstrom
+	to offer
```

Comment:
To stop players with a jump drive from breaking this mission:
```suggestion
	to offer
		not "outfit: Jump Drive"
```

Reply:
I thought of this, hence the dialogue that occurs if you reach systems you can only get to by JD from where the mission offers. It might make sense though if once the main campaign is completed, people become aware of JDs and would hence notice them, so maybe if the main plot is complete and you have a JD it shouldn't offer.